### DISCUSSION TITLE
Storybook -  "register" is null

### DISCUSSION THREAD

**Discussion by ErkanKabil** [react-hook-form/react-hook-form#6447] [Q&A]
### Version Number

7.10.1

### Codesandbox/Expo snack

https://github.com/ErkanKabil/storybook-register-null

I use register with useContextForm() on input component. But when i used Storybook, its set error 'register' is null. 
Then i used FormProvider with useForm. Its works fine. But...
When i change the addons control, its set error like;

![image](https://user-images.githubusercontent.com/58916013/131978510-d0745095-857f-4eda-90a2-6c4c3349447f.png)

Git Repo: https://github.com/ErkanKabil/storybook-register-null

**Comment by bluebill1049** [comment]
This is not an issue with react hook form, check your setup and see if react hook form's context is there.

**Reply by ErkanKabil** [reply]
We tried every way. There is no problem when we using old version of react-hook-form ( we use ref for input register ). But now this is set error.

**Reply by bluebill1049** [reply]
old version of react-hook-form as v6? context API remained the same tho, nothing is changed since then.

**Reply by ErkanKabil** [reply]
Could you please clone the project and examine it? If I don't use register, there is no problem, but when I add register, it gives the error in the photo.

**Reply by bluebill1049** [reply]
could someone else give a test? I am busy with other stuff at the moment. This is definitely not a context issue with hook form but setup.

**Reply by bluebill1049** [reply]
if you reproduce this issue with the context in a codesandbox then I can investigate this issue.

**Comment by bluebill1049** [comment]
this is an old issue: https://github.com/react-hook-form/react-hook-form/issues/4449 not sure if this helps.
